api/managementpb/backup/locations.proto

```diff
@@ -59,7 +59,7 @@ message Location {
   string description = 3;
   oneof config {
     PMMClientLocationConfig pmm_client_config = 4;
-    PMMServerLocationConfig pmm_server_config = 5;
+    PMMServerLocationConfig pmm_server_config = 5 [deprecated = true];
```

Comment:
I think we can just drop it. We still in tech preview, so we can beak API.

Reply:
I've tried to remove unnecessary proto messages, make several renamings, but faced internal compiler error. Let's make futher changes in separate PRs, otherwise this PR is going to became endless.